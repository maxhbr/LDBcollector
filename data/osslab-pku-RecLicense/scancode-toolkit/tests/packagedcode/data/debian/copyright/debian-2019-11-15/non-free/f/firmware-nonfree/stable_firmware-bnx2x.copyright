The binary firmware may be downloaded from
https://git.kernel.org/cgit/linux/kernel/git/firmware/linux-firmware.git

Copyright (c) 2007-2010 Broadcom Corporation

This file contains firmware data derived from proprietary unpublished
source code, Copyright (c) 2007-2009 Broadcom Corporation.

Permission is hereby granted for the distribution of this firmware data
in hexadecimal or equivalent format, provided this copyright notice is
accompanying it.

