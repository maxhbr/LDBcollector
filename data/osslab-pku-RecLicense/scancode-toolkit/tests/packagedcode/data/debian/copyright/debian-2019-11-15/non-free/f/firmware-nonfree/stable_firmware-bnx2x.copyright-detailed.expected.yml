primary_license:
declared_license:
license_expression: proprietary-license
copyright: |
  Copyright (c) 2007-2010 Broadcom Corporation
  Copyright (c) 2007-2009 Broadcom Corporation
matches:
  - score: '87.5'
    start_line: 6
    end_line: 11
    matcher: 3-seq
    rule_length: 35
    matched_length: 35
    match_coverage: '100.0'
    rule_relevance: 100
    identifier: proprietary-license_599.RULE
    license_expression: proprietary-license
    is_license_text: no
    is_license_notice: yes
    is_license_reference: no
    is_license_tag: no
    is_license_intro: no
    matched_text: |
      This file contains firmware data derived from proprietary unpublished
      source code, Copyright ([c]) [2007]-[2009] [Broadcom] [Corporation].

      Permission is hereby granted for the distribution of this firmware data
      in hexadecimal or equivalent format, provided this copyright notice is
      accompanying it.
